ibmcloud/README.md

```diff
@@ -314,18 +405,20 @@ After `terraform apply` completes the `cloud-api-adaptor` process will run on th
 
 ### Deploy the nginx pod and sniff test nginx
 
-A Terraform template that will deploy an nginx pod to the Kubernetes cluster is available in [ibmcloud/terraform/run-nginx-demo](./terraform/run-nginx-demo).
+A Terraform configuration that will deploy an nginx pod to the Kubernetes cluster is available in [ibmcloud/terraform/run-nginx-demo](./terraform/run-nginx-demo). This configuration will also check the nginx peer pod virtual server instance has been successfully created.
 
-Create a `terraform.tfvars` file in the [template directory](./terraform/run-nginx-demo) for this Terraform template on your `development machine`. The `terraform.tfvars` file should look like this
+Create a `terraform.tfvars` file in the [configuration directory](./terraform/run-nginx-demo) for this Terraform configuration on your `development machine`. The `terraform.tfvars` file should look like this
 
 ```
 ibmcloud_api_key = "<your API Key>"
 cluster_name = "<cluster name>"
+vpc_name = "<name of your VPC>" OR vpc_id = "<ID of your VPC>"
```

Comment:
I think `podvm_image_id` , or `podvm_image_id` is a required field for this step as well?